addons/point_of_sale/views/pos_config_view.xml

```diff
@@ -527,6 +527,18 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box" title="Operation types show up in the Inventory dashboard.">
+                            <div class="o_setting_right_pane">
+                                <label for="picking_shipping_type_id" string="Shipping Operation Type"/>
+                                <div class="text-muted">
+                                    Operation type used to record product shipping <br/>
+                                    Products will be taken from the default source location of this operation type
+                                </div>
+                                <div class="content-group mt16">
+                                    <field name="picking_shipping_type_id" required="1" domain="[('company_id', '=', company_id)]"/>
```

Comment:
```suggestion
                                    <field name="picking_shipping_type_id"/>
```

Since it is already given at https://github.com/odoo/odoo/pull/58506/files#diff-d14061cb0b4759aaac1b45637b679e84R224